0_web_functions.R

```diff
@@ -56,14 +56,14 @@ save_if_exists <- function(df, portfolio_name_, save_name, csv_or_rds = "rds") {
 }
 
 set_webtool_paths <- function() {
-  project_location <<- paste0(working_location, "working_dir/")
-
-  log_path <<- paste0(project_location, "00_Log_Files")
-  par_file_path <<- paste0(project_location, "10_Parameter_File")
-  raw_input_path <<- paste0(project_location, "20_Raw_Inputs")
-  proc_input_path <<- paste0(project_location, "30_Processed_Inputs")
-  results_path <<- paste0(project_location, "40_Results")
-  outputs_path <<- paste0(project_location, "50_Outputs")
+  project_location <<- file.path(working_location, "working_dir")
```

Comment:
Nit: You may also use `fs::path()`. I don't mind depending on fs -- it's well developed an has 0 dependencies

<https://cran.r-project.org/web/packages/fs/index.html>

Reply:
I don't know if there's really any advantage to that, but I could redo it tomorrow with `fs::path()`

Reply:
I think there is no advantage, other than the shorter name.